src/harness/util.py

```diff
@@ -376,3 +375,40 @@ def addGrantIdsToRequests(grant_ids, requests):
     requests: (list) list of requests, containing dictionaries.
   """
   addIdsToRequests(grant_ids, requests, 'grantId')
+
+def _orderAttributes(obj):
+    if isinstance(obj, dict):
+        return sorted((k, _orderAttributes(v)) for k, v in obj.items())
+    if isinstance(obj, list):
+        return sorted(_orderAttributes(x) for x in obj)
+    else:
+        return obj
+
+# Test the Compare Dict function which is used in FAD.1 to compare PPA record and Esc records in dump file with injected object
+def compareDict(first_dict, second_dict):
```

Comment:
Isn't this already handled by the `==` operator? See https://stackoverflow.com/a/5635309